pkg/controllers/user/logging/utils/daemonset.go

```diff
@@ -151,11 +192,19 @@ func newFluentdDaemonset(name, namespace, clusterName string) *v1beta2.DaemonSet
 									Name:      "projectlogging",
 									MountPath: "/fluentd/etc/config/project",
 								},
+								{
+									Name:      "clustercustomlogconfig",
+									MountPath: "/fluentd/etc/config/customer/cluster",
+								},
+								{
+									Name:      "projectcustomlogconfig",
+									MountPath: "/fluentd/etc/config/customer/project",
+								},
 							},
 						},
 						{
 							Name:            loggingconfig.FluentdName,
-							Image:           v3.ToolsSystemImages.LoggingSystemImages.Fluentd,
+							Image:           "micheliac/fluentd:v0.30",
```

Comment:
we should use v3.ToolsSystemImages.LoggingSystemImages.Fluentd

Reply:
yes, will update after built